### DISCUSSION TITLE
Setting up x.509 client side mTLS for IoT device

### DISCUSSION THREAD

**Discussion by suyambutriveni** [xipki/xipki#279] [Q&A]
Hi Team,
Is there any documentation or blog or some kind of quick start detailing on how to setup xipki server for provisioning X.509 client side certification for IoT device and validating the certificate on server side MQTT broker for mTLS? @xipki

**Comment by xipki** [comment]
In general, the IoT device may apply for new certificate with any protocol supported by XiPKI protocol gateway. The easiest way is to use the EST protocol or XiPKI-own RESTful API.
